### PR TITLE
YAML-Updates-2-23-2024

### PR DESCRIPTION
**Pull request by raul-marquez-csa**
Issues covered:
[[TC-TMP-2.1] MaxMeasuredValue range has to be updated as per Spec Matter_1.3](https://github.com/project-chip/matter-test-scripts/issues/158) (TC-TMP-2.1)
[[1.3] DLOG Cluster testcase has to be updated as per the latest Test-plan. Matter_1.3](https://github.com/project-chip/matter-test-scripts/issues/162) (TC-DLOG-2.1, TC-DLOG-2.2, TC-DLOG-3.1)
[[TC-CNET-4.12] Script has to be updated as per VS changes #167](https://github.com/project-chip/matter-test-scripts/issues/167) (TC-CNET-4.12)
[[Test_TC_PWRTL_1_1] - Script should be updated by adding test steps to read the AcceptedCommandList and GeneratedCommandList attribute #168
](https://github.com/project-chip/matter-test-scripts/issues/168) (TC-PWRTL-1-1)
[[1.3][TC-EEVSE-2.1] YAML script has to be updated as per spec with removing the Max range constraint #169
](https://github.com/project-chip/matter-test-scripts/issues/169) (TC-EEVSE-2.1)
[[TC-GRPKEY -2.1, 2.2] Script has to be updated as per Test Plan #171
](https://github.com/project-chip/matter-test-scripts/issues/171) (TC-GRPKEY -2.1)

### CODE REVIEW

**Comment by raju-apple** [comment]
> @raju-apple ?

Yeah , some of the minor fixes which missed the TH cutoff date , but we would want to get these on the branch before the final official TH version comes out after SVE

**Comment by raju-apple** [comment]
@woody-apple  Yeah , some of the minor fixes which missed the TH cutoff date , but we would want to get these on the branch before the final official TH version comes out after SVE

File: src/app/tests/suites/certification/Test_TC_CNET_4_12.yaml [hunk 1]
```diff
@@ -180,34 +180,13 @@ tests:
       PICS: CNET.S.C06.Rsp
```
Comment (KishokG) [inline src/app/tests/suites/certification/Test_TC_CNET_4_12.yaml:180]: @raul-marquez-csa TC-CNET-4.12: Script has to be updated from Step 8 to 20 as per Verification steps. We have a sample app issue in Step 8 to 20 so marked as red color. But the Verification steps are correct.

Comment (raul-marquez-csa) [reply src/app/tests/suites/certification/Test_TC_CNET_4_12.yaml:190]: Updated

**Review by raju-apple — APPROVED** [review]

**Pull request merged**